js/script.js

```diff
@@ -1 +1,6 @@
-console.log ('hello');
\ No newline at end of file
+"use strict"
+let user = '',
+     userName = 'Назар',
+     colorGlas = 'синій';
+    user = userName;
+console.log (user + ' має ' + colorGlas + ' колір очей ');
```

Comment:
Можна використати шаблонні строки `` щоб не використовувати конкантенацію
```suggestion
console.log (`${user.firstName} має ${user.eyeColour} колір очей`);
```